examples/analog_hamiltonian_simulation/04_Maximum_Independent_Sets_with_Analog_Hamiltonian_Simulation.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>Maybe adding markers where the control points for the waveform are could be interesting. Feel free to ignore if you think this does not add to the plot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>